src/compiler/Parser/ParsedHierarchy/Declaration/ConstantDeclaration.ts

```diff
@@ -32,6 +32,10 @@ export class ConstantDeclaration extends ParsedObject {
     }
   }
 
+  get typeName(): string {
+    return "CONST";
```

Comment:
It's different from upstream, but it shouldn't make any difference I guess? Any reason for the change?

Reply:
It weirded me out that the type did not look like its declaration (it's the case for the ones that were already defined) and created a special exception when playing with the parsedHierrarchy tree.